### PR TITLE
Theme homepage

### PR DESCRIPTION
**Pull request by muhajirdev**
### Summary
- homepage theme

### Screenshots

| dark | light | sepia |
| ------ | ------ | -----|
| image | image | image
|

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "typescript.format.enable": false,
     "javascript.format.enable": false,
     "cssVariables.lookupFiles": [
-        "src/styles/theme.scss"
+        "src/styles/theme.scss",
+        "src/styles/themes/dark.scss", // include one of the theme file
```
Comment (tifa2UP) [inline .vscode/settings.json:16]: thank you for catching this! let's include the light one since it's the default

File: public/icons/Q.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
+<svg width="25" height="24" viewBox="0 0 25 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```
Comment (tifa2UP) [inline public/icons/Q.svg:1]: One alternate approach is to create an Icon component in the dls (if there's isn't one already). 

This icon component is theme aware, meaning that it passes the fill color based on the theme. 

You can set the fill color in scss using the following syntax: 

```
svg {
	path {
		fill: (color)
	}
}

```

File: src/components/Navbar/Navbar.module.scss [hunk 3]
```diff
@@ -21,6 +21,7 @@
   flex-direction: row;
   justify-content: space-between;
   height: 100%;
+  color: var(--color-text-default);
```
Comment (tifa2UP) [inline src/components/Navbar/Navbar.module.scss:24]: yup, this is the desired approach. Thank you Muhajir!

**Comment by mmahalwy** [comment]
#### 📝Changed routes:


| [**/index.module**](https://quran-com-frontend-next-k0y4316d1-qurancom.vercel.app/index.module) |
|:-:|
|<a href="https://quran-com-frontend-next-k0y4316d1-qurancom.vercel.app/index.module">Screenshot of /index.module</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/3k8MBwDRBlS-P15BRMPlaewHCZjXtY80f_m68ey5oSVkhLu3V6PVvaNCAiMUPBfthHcFSfSg5L6T7FtNcH6oVw.png">(view full size)</a> |







Commit a2a840550d4e5f7d37fed33f020aee7d4234db04 (https://quran-com-frontend-next-k0y4316d1-qurancom.vercel.app).

File: public/icons/Q.svg (hunk 2 above)
Comment (tifa2UP) [reply public/icons/Q.svg:1]: that works :D

**Review by tifa2UP — APPROVED** [review]

**Pull request merged**